teuthology/task/install/__init__.py

```diff
@@ -252,8 +252,10 @@ def install(ctx, config):
     finally:
         remove_packages(ctx, config, package_list)
         remove_sources(ctx, config)
-        if project == 'ceph':
-            purge_data(ctx)
+        # Not needed when the VM is going to be destroyed
```

Comment:
Why do we need this change?
How is it related to the title of the PR: 'drop ceph-deploy overrides'?

Reply:
@kshtsk Please read the commit description. Do you want me to make a separate PR for the second commit?

Reply:
I'm only commenting it out for now, since it needs to be removed upstream.

Reply:
@smithfarm I'm not sure that purging data is related to 'drop ceph-deploy', I thought it is related more to 'install' task itself, which installing rpms from buildpackages repo server.

Reply:
No need in separate PR if you're not squashing it.

Reply:
@kshtsk It'll be clearer if you look at each commit separately.